### PR TITLE
Teams component

### PR DESCRIPTION
**Pull request by MaxwellHouston**
Created Teams component, Added header and data table with sample input.
Created CSS file and rendering tests.

### CODE REVIEW

File: view/src/components/teams/Teams.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
+import { DataTable } from '../dataTable/DataTable';
+
 export const Teams = () => {
 
+    const testData = [{"row_id":"ARI","team_id":"ARI","qb":"344.04","rb":"348.60","wr":"436.52","te":"101.80"},{"row_id":"ATL","team_id":"ATL","qb":"222.92","rb":"313.50","wr":"223.80","te":"158.80"},{"row_id":"BAL","team_id":"BAL","qb":"341.78","rb":"270.30","wr":"304.60","te":"201.10"},{"row_id":"BUF","team_id":"BUF","qb":"411.84","rb":"277.60","wr":"477.90","te":"125.20"},{"row_id":"CAR","team_id":"CAR","qb":"245.80","rb":"260.00","wr":"302.30","te":"48.80"},{"row_id":"CHI","team_id":"CHI","qb":"227.48","rb":"262.20","wr":"315.00","te":"109.70"}]
```
Comment (bennidhamma) [inline view/src/components/teams/Teams.js:5]: Can you move this to the test file and access as a prop?

File: view/src/components/teams/Teams.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { render, screen, cleanup } from '@testing-library/react';
+import { Teams } from './Teams';
+
+afterEach(() => {
+    cleanup();
+});
+
+describe('Elements render properly', () => {
+
+    test('Header renders', () => {
```
Comment (bennidhamma) [inline view/src/components/teams/Teams.test.js:10]: These tests are a bit light. Can you do anything a bit more interesting? Like maybe check that the row length is what you would expect it to be? Or if that is hard to test, maybe just that some of the cell data is present?

**Review by bennidhamma — CHANGES_REQUESTED** [review]
Best to not leave test data in production code...

File: view/src/components/teams/Teams.test.js (hunk 2 above)
Comment (MaxwellHouston) [reply view/src/components/teams/Teams.test.js:10]: So I have tests written for the DataGrid component that test that it handles inputs correctly. Should I also test it here with this specific type of input?

File: view/src/components/teams/Teams.js (hunk 1 above)
Comment (MaxwellHouston) [reply view/src/components/teams/Teams.js:5]: Sounds good. My overall plan was to create the barebones components then come back and add the data fetching and CSS. should I not submit pull requests with only the blank component? Or just delete the test data once I see it works.

**Pull request merged**